jhipster-jdl.jdl

```diff
@@ -225,21 +237,22 @@ relationship OneToMany {
     CompanionDiagnosticDevice to FdaSubmission
     Consequence to Alteration
     FdaSubmissionType to FdaSubmission{type}
-    FdaSubmission to DeviceUsageIndication
-	Alteration to DeviceUsageIndication
 	Alteration{referenceGenomes} to AlterationReferenceGenome
 	CancerType to DeviceUsageIndication
-	Drug to DeviceUsageIndication
+    Gene to DeviceUsageIndication
     Drug{brands} to DrugBrand
 }
 
 relationship ManyToMany {
     Alteration to Gene
     CompanionDiagnosticDevice to SpecimenType
     ClinicalTrialsGovCondition to CancerType
+    DeviceUsageIndication to Alteration
+    DeviceUsageIndication to Drug
+    DeviceUsageIndication to FdaSubmission
```

Comment:
Updated DeviceUsageIndication relationships
- Drugs: allow for combination druugs
- FdaSubmission: allow for multiple PMAs where evidence is retrieved from
- Alteration: multiple alterations in association